### PR TITLE
Production fixes

### PR DESCRIPTION
**Pull request by jdesboeufs**
- Add required files for `foreman` process management
- Enable CORS on all API routes (to be fixed)

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -0,0 +1 @@
+web: npm test
```
Comment (MattiSG) [inline Procfile]: Shouldn't that be `npm start`?

Comment (jdesboeufs) [inline Procfile]: Yes!

**Comment by MattiSG** [comment]
If CORS should be enabled only on specific routes, then we should enable it in sgmap/ludwig-api, right?

**Comment by MattiSG** [comment]
We actually already had `cors` enabled on `/public` routes in `ludwig-api`, simply not using this route to `POST`…

Though, creating a test that way without being logged redirects to an frozen “loading” page crashing due to `401`s. Need to find a proper workflow for that.

**Comment by MattiSG** [comment]
Merged subset in f730ed2.

Dropping f13a119 to use public routes.

**Pull request closed without merging**